bulk_insert.go

```diff
@@ -91,7 +91,7 @@ func insertObjSet(db *gorm.DB, objects []interface{}, excludeColumns ...string)
 		insertOption = strVal
 	}
 
-	mainScope.Raw(fmt.Sprintf("INSERT INTO %s (%s) VALUES %s %s",
+	mainScope.Raw(fmt.Sprintf("REPLACE INTO %s (%s) VALUES %s %s",
```

Comment:
I don't think it's a good idea to assume that everyone using this package want's to use `REPLACE`. This is indeed different from `INSERT` so this would be a breaking change for a lot of cases.

If you're interested in adding new functionality, please feel free to add new methods that doesn't introduce breaking changes in the existing API.

Reply:
sorry, this PR is by mistake

Reply:
this was only my experiment. Maybe I will make it more generic.